.sonarwhalrc

```diff
@@ -1,11 +1,12 @@
 {
     "connector": {
-        "name": "chrome",
+        "name": "jsdom",
```

Comment:
Any reason why we are changing this?

Reply:
No, I though `jsdom` was the default hehehe